city_scrapers/spiders/il_corrections.py

```diff
@@ -0,0 +1,160 @@
+import re
+from datetime import datetime
+from io import BytesIO, StringIO
+
+import scrapy
+from city_scrapers_core.constants import ADVISORY_COMMITTEE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from pdfminer.high_level import extract_text_to_fp
+from pdfminer.layout import LAParams
+
+
+class IlCorrectionsSpider(CityScrapersSpider):
+    name = "il_corrections"
+    agency = "Illinois Department of Corrections Advisory Board"
+    timezone = "America/Chicago"
+    start_urls = [
+        "https://www2.illinois.gov/idoc/aboutus/advisoryboard/Pages/default.aspx"
+    ]
+
+    def __init__(self):
```

Comment:
It looks like we don't need to set any of these on initialization, and other than `links` we may not need to set any of these directly on the spider itself because it looks like they could be handled as normal variables. Let me know if I'm missing something there though!

Reply:
Based on other commends I'm leaving links and url_base for now